bazel/repository_locations.bzl

```diff
@@ -171,130 +210,155 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "zlib-79baebe50e4d6b73ae1f8b603f0ef41300110aa3",
         # 2019-04-14 development branch
         urls = ["https://github.com/madler/zlib/archive/79baebe50e4d6b73ae1f8b603f0ef41300110aa3.tar.gz"],
+        use_category = ["dataplane"],
     ),
     com_github_jbeder_yaml_cpp = dict(
         sha256 = "77ea1b90b3718aa0c324207cb29418f5bced2354c2e483a9523d98c3460af1ed",
         strip_prefix = "yaml-cpp-yaml-cpp-0.6.3",
         urls = ["https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.3.tar.gz"],
+        use_category = ["dataplane"],
     ),
     com_github_msgpack_msgpack_c = dict(
         sha256 = "433cbcd741e1813db9ae4b2e192b83ac7b1d2dd7968a3e11470eacc6f4ab58d2",
         strip_prefix = "msgpack-3.2.1",
         urls = ["https://github.com/msgpack/msgpack-c/releases/download/cpp-3.2.1/msgpack-3.2.1.tar.gz"],
+        use_category = ["dataplane"],
     ),
     com_github_google_jwt_verify = dict(
         sha256 = "d422a6eadd4bcdd0f9b122cd843a4015f8b18aebea6e1deb004bd4d401a8ef92",
         strip_prefix = "jwt_verify_lib-40e2cc938f4bcd059a97dc6c73f59ecfa5a71bac",
         # 2020-02-11
         urls = ["https://github.com/google/jwt_verify_lib/archive/40e2cc938f4bcd059a97dc6c73f59ecfa5a71bac.tar.gz"],
+        use_category = ["dataplane"],
     ),
     com_github_nodejs_http_parser = dict(
         sha256 = "8fa0ab8770fd8425a9b431fdbf91623c4d7a9cdb842b9339289bd2b0b01b0d3d",
         strip_prefix = "http-parser-2.9.3",
         urls = ["https://github.com/nodejs/http-parser/archive/v2.9.3.tar.gz"],
+        use_category = ["dataplane"],
     ),
     com_github_pallets_jinja = dict(
         sha256 = "db49236731373e4f3118af880eb91bb0aa6978bc0cf8b35760f6a026f1a9ffc4",
         strip_prefix = "jinja-2.10.3",
         urls = ["https://github.com/pallets/jinja/archive/2.10.3.tar.gz"],
+        use_category = ["build"],
     ),
     com_github_pallets_markupsafe = dict(
         sha256 = "222a10e3237d92a9cd45ed5ea882626bc72bc5e0264d3ed0f2c9129fa69fc167",
         strip_prefix = "markupsafe-1.1.1/src",
         urls = ["https://github.com/pallets/markupsafe/archive/1.1.1.tar.gz"],
+        use_category = ["build"],
     ),
     com_github_tencent_rapidjson = dict(
         sha256 = "a2faafbc402394df0fa94602df4b5e4befd734aad6bb55dfef46f62fcaf1090b",
         strip_prefix = "rapidjson-dfbe1db9da455552f7a9ad5d2aea17dd9d832ac1",
         # Changes through 2019-12-02
         urls = ["https://github.com/Tencent/rapidjson/archive/dfbe1db9da455552f7a9ad5d2aea17dd9d832ac1.tar.gz"],
+        use_category = ["dataplane"],
```

Comment:
Is this anywhere on the controlplane still?

Reply:
Not last time I checked, seems to be dataplane use only.